client/my-sites/site-settings/disconnect-site/troubleshoot.jsx

```diff
@@ -5,6 +5,7 @@
 import React from 'react';
 import { connect } from 'react-redux';
 import { localize } from 'i18n-calypso';
+import Gridicon from 'gridicons';
```

Comment:
(behold the king of the nits)

ABC police 🔤  - letters come before curly braces, so the `Gridicon` import should go below the `React` one (or above it, depending on how 🔤 -fanatic you are 😉 ).

Reply:
2bbef65